docs(35/centos) improve the CentOS installation guide (#1367)
Kong/docs.konghq.com: app/enterprise/0.35-x/deployment/installation/centos.md

## Installation Steps

> This installation guide assumes:

Reviewer: ```suggestion
### Prerequisites

This installation guide assumes:
```

## Securely copy the license file to the CentOS system ##
$ scp kong-se-license.json <centos username>@192.168.91.109:~

4. Install Kong Enterprise (Note your version will be different based on when you obtained the rpm)

$  sudo yum install kong-enterprise-edition-0.35-1.el7.noarch.rpm

5. Install Postgresql 

$ sudo yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm

$ sudo yum install postgresql95 postgresql95-server

6. Initialize Postgresql database

$ sudo /usr/pgsql-9.5/bin/postgresql95-setup initdb

7. Start Postgres and Enable automatic start

$ sudo systemctl enable postgresql-9.5
$ sudo systemctl start postgresql-9.5

8. Create Kong database and user. NOTE: Make sure the username and password for the Kong Database are kept safe. We have used a simple example below for illustration purposes. 

## switch to postgres user ##
$sudo -i -u postgres

## launch postgresql ##
$psql

> ## Create kong user and database, make kong the owner of the database and set the password of the kong user to 'kong' ##

Reviewer: ```suggestion
**Create Kong user and database, make Kong the owner of the database and set the password of the Kong user to 'kong'**
```

> $CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';

Reviewer: ```suggestion
$ CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
```

> ## exit from psql ##

Reviewer: ```suggestion
**exit from psql**
```

> $\q

Reviewer: ```suggestion
`$ \q`
```

> ## return to terminal ##

Reviewer: ```suggestion
**return to terminal**
```

> $ exit

Reviewer: ```suggestion
`$ exit`
```

> 9. Modify postgres conf to allow password authentication with md5. 

Reviewer: ```suggestion
9. Modify the Postgres configuration to allow password authentication with MD5.
```

> ## Under IPv4 local connections: replace the "ident" entry with "md5"

Reviewer: ```suggestion
**Under IPv4 local connections: replace the `"ident"` entry with `"md5"`**
```

> $ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf 

Reviewer: ```suggestion
`$ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf`
```

> ## restart postgresql ##

Reviewer: ```suggestion
**restart postgresql**
```

> $ sudo systemctl restart postgresql-9.5

Reviewer: ```suggestion
`$ sudo systemctl restart postgresql-9.5`
```

> 10. Copy the license file from your home directory to the /etc/kong directory. Kong will look for the licence here. 

Reviewer: Instead of "Kong will look for the license", could we use something less anthropomorphic such as "Kong will use the license located here."

```suggestion
10. Copy the license file from your home directory to the `/etc/kong` directory. Kong will look for the license here. 
```

> $ sudo cp kong-se-license.json /etc/kong/license.json

Reviewer: ```suggestion
`$ sudo cp kong-se-license.json /etc/kong/license.json`
```

> 11. Modify kong.conf file and update the postgres database password to the password you used when you created the "kong" postgresql user. 

Reviewer: ```suggestion
11. Modify the `kong.conf` file and update the Postgres database password to the password you used when you created the Postgres user named **kong**. 
```

> ## Make a copy of the default conf file

Reviewer: ```suggestion
**Make a copy of the default conf file**
```

> $ cp /etc/kong/kong.conf.default /etc/kong/kong.conf

Reviewer: ```suggestion
`$ cp /etc/kong/kong.conf.default /etc/kong/kong.conf`
```

> # Uncomment and add 'kong' to pg_password line

Reviewer: ```suggestion
**Uncomment and add 'kong' to pg_password line**
```

> $ sudo vi [/path/to/kong.conf]

Reviewer: ```suggestion
`$ sudo vi [/path/to/kong.conf]`
```

> 12. Seed the superuser for initial startup. This step is not mandatory but if you want to add security and Role Based Access Control to your environment, it is best to seed the super-user at this step and make note of the password for when you need it later. 

Reviewer: ```suggestion
12. Seed the **Super Admin** for initial startup. This step is not mandatory, but if you want to add security and Role-Based Access Control (RBAC) to your environment, it is best to seed the **Super Admin** at this step and make note of the password for when you need it later. 
```

> # Create Environment Variable with super-admin password (This will be used during migrations to seed the initial super-admin password)

Reviewer: ```suggestion
**Create an environment variable with the _Super Admin_ password (that will be used during migrations to seed the initial _Super Admin_ password)**
```

> $ export KONG_PASSWORD=<password-only-you-know>

Reviewer: ```suggestion
`$ export KONG_PASSWORD=<password-only-you-know>`
```

13. Run migrations to prepare the Kong DB

> $ kong migrations bootstrap -c /etc/kong/kong.conf

Reviewer: ```suggestion
`$ kong migrations bootstrap -c /etc/kong/kong.conf`
```

14. Start kong

> $ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf

Reviewer: ```suggestion
`$ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf`
```

15. Verify Kong installation is working

> > curl -i -X GET --url http://localhost:8001/services

Reviewer: ```suggestion
`curl -i -X GET --url http://localhost:8001/services`
```

> # You should receive a HTTP/1.1 200 OK message. If you do, you are ready to move on. As a Next Step, consider the following tasks:

Reviewer: ```suggestion
**You should receive a HTTP/1.1 200 OK message. If you do, you are ready to move on. As a Next Step, consider the following tasks:**
```

1. Turn on RBAC and understand the RBAC model within Kong. 
2. Turn on the Admin GUI Portal to allow you to administer Kong through the UI. 
3. Create a sample Workspace to understand how Kong can be segregated for use between teams. 
4. Create Services, Routes, and Consumers for a given workspace. 
5. Add Plugins to inject functionality to the API calls that are being proxied through Kong. For example, Add Rate-Limiting. 

> 2. 

Reviewer: ```suggestion
```

## Installation Steps

### Prerequisites

This installation guide assumes:

> * You have a CentOS 7 system with root access. 

Reviewer 2: Change root to root equivalent.

> * You can SSH to that system to configure it. 

Reviewer 2: You can SSH to the system
